jobs/JGLOBAL_ATMOS_ANALYSIS_CALC

```diff
@@ -28,16 +28,19 @@ export GPREFIX="${GDUMP}.t${gcyc}z."
 export APREFIX="${RUN}.t${cyc}z."
 export GPREFIX_ENS="${GDUMP_ENS}.t${gcyc}z."
 
-RUN=${rCDUMP} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS
+RUN=${rCDUMP} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_OBS:COM_OBS_TMPL
 
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS COM_ATMOS_RESTART
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx 
```

Comment:
@mingshichen-noaa 
This line is causing the failure.  Please fix

Reply:
@aerorahul 
 
After I checked codes in  jobs/JGLOBAL_ATMOS_ANALYSIS_CALC from mingshichen-noaa:feature/jgdas_enkf_select_obs_com_in_out,  the codes had been modified as follows:
 30
 31 RUN=${rCDUMP} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
 32     COMIN_OBS:COM_OBS_TMPL
 33
 34 YMD=${PDY} HH=${cyc} declare_from_tmpl -rx
 35     COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
 36     COMIN_ATMOS_RESTART:COM_ATMOS_RESTART_TMPL
 37
 38 RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
 39     COMIN_OBS_PREV:COM_OBS_TMPL \
 40     COMIN_ATMOS_HISTORY_PREV:COM_ATMOS_HISTORY_TMPL
 41